tests/conftest.py

```diff
@@ -35,14 +35,9 @@
 from .helpers import SecuredClient
 
 
-@pytest.fixture(scope="session")
-def api_key_header():
-    return {API_KEY_HEADER_NAME: DEFAULT_API_KEY}
-
-
 @pytest.fixture(scope="session")
 def client():
-    with TestClient(app) as c:
+    with TestClient(app, raise_server_exceptions=False) as c:
```

Comment:
Setting `raise_server_exceptions` to `False` is making the tests working the same that normal behavior. The exceptions raised by the server will now return a 500 response for example (instead of returning that exception and not returning a response).

Reply:
Be careful, this will by-pass other errors than 500 (Remember 401 exception in tests)